src/contexts/auth/AuthContext.tsx

```diff
@@ -236,17 +236,17 @@ const AuthProvider = memo(({ children }: Props) => {
 
   return (
     <Fragment key={shouldDisplayUniversalLoader.toString()}>
-      <div style={{ display: shouldDisplayUniversalLoader ? 'none' : undefined }}>
-        <ErrorBoundary>
-          <Suspense fallback={<Loader />}>
-            <AuthStateContext.Provider value={authState}>
-              <AuthActionsContext.Provider value={authActions}>
-                <Web3WalletProvider>{children}</Web3WalletProvider>
-              </AuthActionsContext.Provider>
-            </AuthStateContext.Provider>
-          </Suspense>
-        </ErrorBoundary>
-      </div>
+      <ErrorBoundary>
+        <Suspense fallback={<FullPageLoader />}>
+          <AuthStateContext.Provider value={authState}>
+            <AuthActionsContext.Provider value={authActions}>
+              <Web3WalletProvider>
+                {shouldDisplayUniversalLoader ? null : children}
```

Comment:
main change is waiting to render children until auth state is confirmed. More details in PR description